test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -177,84 +165,81 @@ public void GetTestsShouldNotReturnBaseTestMethodsFromAnotherAssemblyByConfigura
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.IsTrue(
+        Verify(tests is not null);
+        Verify(
             tests.All(t => t.TestMethod.Name != "BaseTestMethod"),
             "DummyDerivedFromRemoteTestClass inherits DummyRemoteBaseTestClass from different assembly. BestTestMethod from DummyRemoteBaseTestClass should not be discovered when RunSettings MSTestV2 specifies EnableBaseClassTestMethodsFromOtherAssemblies = false.");
     }
 
-    [TestMethod]
     public void GetTestsShouldNotReturnHiddenTestMethods()
     {
         SetupTestClassAndTestMethods(isValidTestClass: true, isValidTestMethod: true, isMethodFromSameAssembly: true);
         TypeEnumerator typeEnumerator = GetTypeEnumeratorInstance(typeof(DummyHidingTestClass), Assembly.GetExecutingAssembly().FullName);
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.AreEqual(
-            1,
+        Verify(tests is not null);
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "BaseTestMethod"),
             "DummyHidingTestClass declares BaseTestMethod directly so it should always be discovered.");
-        Assert.AreEqual(
-            1,
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "DerivedTestMethod"),
             "DummyHidingTestClass declares BaseTestMethod directly so it should always be discovered.");
-        Assert.IsFalse(
+        Verify(!
             tests.Any(t => t.TestMethod.DeclaringClassFullName == typeof(DummyBaseTestClass).FullName),
             "DummyHidingTestClass hides BaseTestMethod so declaring class should not be the base class");
```

Comment:
Please move this arg to be a comment before the `Verify`. It's currently going in the `expression` param which we don't want to be filled by "user".